### PR TITLE
created Section Title Component

### PR DESCRIPTION
**Pull request by Fredmpk**
## Checklist

Confirm you have completed the following actions prior to submitting this PR.

- [x] There is an existing issue report for this PR.
- [x] I have created a feature branch.
- [x] My changes have been committed.
- [x] I have pushed my changes to the branch.

## Title

created Section Title Component

## Description

I created a component for the section title, it's possible to pass a text and show or hide the See All - link through the props.

### CODE REVIEW

File: src/components/section-title.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+type Props = {
+  text: string;
+  ShowSeeAll?: boolean;
+};
+
+export function SectionTitle({ text, ShowSeeAll }: Props) {
+  return (
+    <div className="flex justify-between">
+      <span className="text-white">{text}</span>
```
Comment (dnmct) [inline src/components/section-title.tsx]: Font should be medium weight

File: src/components/section-title.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      {ShowSeeAll && (
+        <Link to="/Placeholder" className="text-yellow">
+          <div className="flex">
+            <span>See All</span>
```
Comment (dnmct) [inline src/components/section-title.tsx]: Text is too big

File: src/components/section-title.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <svg
```
Comment (dnmct) [inline src/components/section-title.tsx]: Use a heroicons arrow

File: src/components/section-title.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  ShowSeeAll?: boolean;
+};
+
+export function SectionTitle({ text, ShowSeeAll }: Props) {
+  return (
+    <div className="flex justify-between">
+      <span className="text-white">{text}</span>
+      {ShowSeeAll && (
+        <Link to="/Placeholder" className="text-yellow">
+          <div className="flex">
```
Comment (dnmct) [inline src/components/section-title.tsx]: gap between text and arrow is missing

File: src/components/section-title.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+
+type Props = {
+  text: string;
+  ShowSeeAll?: boolean;
+};
+
+export function SectionTitle({ text, ShowSeeAll }: Props) {
+  return (
+    <div className="flex justify-between">
```
Comment (dnmct) [inline src/components/section-title.tsx]: also align on the vertical axis here

**Review by dnmct — DISMISSED** [review]

**Review by louziferr — APPROVED** [review]
good job !

**Pull request merged**